### ISSUE TITLE
Faeture request: Toggle view between plain Text and HTML

### ISSUE THREAD

**Issue by webratte** [k9mail/k-9#1397]
Title says all I hope ;-)

I also hope this issue doesn’t exist yet.

**Comment by philipwhiuk** [comment]
See also #906 deals with making plaintext an option by default.

**Comment by francwalter** [comment]
> @webratte  Title says all I hope

I hope the typo in the title "Faeture..." doesn't say this will never be implemented ;)

**Comment by PokerFace8** [comment]
Hey, I would like to work on this. Could you help me get started on this(as in any pointers and/or specific instructions) ?

**Comment by Valodim** [comment]
I don't think this issue requires any handholding. Feel free to ask concrete questions if you have some :)

**Comment by PokerFace8** [comment]
Since I am a beginner I am finding it difficult to understand the code base. Which files do I start working on?

**Comment by Valodim** [comment]
We don't have capacity to micromanage contributors at that level, sorry. Please get comfortable with Android Studio to find your way around the codebase on your own.

**Comment by PokerFace8** [comment]
Okay. Got it :)

**Comment by shaunaqnarindra** [comment]
Hi valodim,

Just to be sure, you want us to toggle the view while composing our message right?

Regards

**Comment by philipwhiuk** [comment]
No, this is about reading messages.

**Comment by shaunaqnarindra** [comment]
@PokerFace8 Not a problem :) I belive I can contribute till the issue is open

**Comment by shaunaqnarindra** [comment]
@PokerFace8 Not a problem :), I believe I can contribute to the issue till it is open.

**Comment by PokerFace8** [comment]
@Valodim please check #2334 . I have corrected it to use the proper text but have some doubts which are addressed there.

**Comment by McLP2** [comment]
I think this is also relevant for security, as html mails can be abused to steal other encrypted mails. Keyword "efail". I really want to disable them, just in case.

**Comment by PendragonII** [comment]
Yes, HTML to Plain Text should be added if only because it is a security feaure (more on that here: https://www.eff.org/deeplinks/2018/05/pgp-and-efail-frequently-asked-questions#html )

**Comment by seniorm0ment** [comment]
This is something I'd really like to see.

**Comment by chekovma** [comment]
I actually ran across an issue earlier this year when our marketing team was sending out emails that had different html and text parts on accident.  When a new person took over the job they did not know to change both parts and it was not done automatically so the text part contained old links and news while the html part was current.
I point this out because I could only diagnose the problem on my desktop email client, it would have been impossible to do on Thunderbird android since there is no way for me to even ever see the text/plain part.  More broadly if I'm on a support call and I'm looking at an email and what I see is not what the person on the other end of the line sees it makes it hard if not impossible to help them.
I saw on the other thread that they were voting on a feature request to add the default view option.  I'd say this toggle is actually more important and it should more properly be treated as a bug instead of a feature request.  The email client as it is now cannot display the content of emails.   That's kind of the point, eh?
